### PR TITLE
add RockPaperScirssors Test

### PR DESCRIPTION
**Pull request by yusufidimaina9989**

### CODE REVIEW

File: tests/rps.test.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    const instance = new RockPaperScissors()
+
+    await instance.connect(getDefaultSigner())
+    const deployTx = await instance.deploy(1)
+    console.log(`RockPaperScirssors contract deployed:  ${deployTx.id}`)
+
+    const callContract = async () => {
+        await instance.methods.main(3n, myAddress, 12n)
+        expect(callContract()).not.Throw
+    }
```
Comment (zhfnjust) [inline tests/rps.test.ts:18]: good work, do mirror fix.
```ts  
const callContract = async () => {
        instance.methods.main(3n, myAddress, 12n)
   }

return   expect(callContract()).not.be.rejected
```

Comment (zhfnjust) [reply tests/rps.test.ts:18]: see demo test: https://github.com/sCrypt-Inc/boilerplate/blob/master/tests/demo.test.ts

Comment (yusufidimaina9989) [reply tests/rps.test.ts:18]: ok

Comment (yusufidimaina9989) [reply tests/rps.test.ts:18]: ok

Comment (yusufidimaina9989) [reply tests/rps.test.ts:27]: done

**Review by xhliu — APPROVED** [review]
👍 well done

**Pull request merged**